scalafix-tests/input/src/main/scala/test/DisableSimple.scala

```diff
@@ -49,7 +59,7 @@ case object Disable {
   }
   val yy = AA.asInstanceOf // OK, no errors
   Option(1).get /* assert: Disable.Option.get
-            ^
+  ^
```

Comment:
Why this change? The caret should point to `get`

Reply:
I use `ctx.index.symbol(tree)` to get symbol during tree traversal. Looking at sources for that method we have ` case Term.Select(_, name @ Name(_)) => symbol(name)` :)

Reply:
We should report only on the `Name`, if you are in a pattern match the best way to do it is with `case index.Symbol(tree: Name) => ...`